templates/home.html

```diff
@@ -42,6 +43,9 @@ <h4 class="mt-4" id="recent-reviews-header">Recent Reviews</h4>
             <span><i class="fa-solid fa-star"></i></span>
             {% endfor %}
             <p class="mt-2">{{ review.description }}</p> 
+            {% for photo in review.photos %}
+            <img class='review_photos' src="{{photo.photo_url}}" alt="review_photo"> 
```

Comment:
When you have a lot of photos this might be a problem. Consider using lazy loading approaches.

Reply:
done